docs/cloud-porfile.md

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
file naming typo

Reply:
updated, also added `mkdocs` and `mkdocs-click` information in readme